spec/tramway/configs/entity_spec.rb

```diff
@@ -18,37 +32,53 @@
     end
 
     let(:expected_route) do
-      Rails.application.routes.url_helpers.public_send("#{name.to_s.pluralize.parameterize.underscore}_path")
+      helper = if route.present?
+                 if route[:namespace].present? && route[:route_method].nil?
+                   "#{route[:namespace]}_#{name.to_s.pluralize.parameterize.underscore}_path"
+                 elsif route[:namespace].present? && route[:route_method].present?
+                   "#{route[:namespace]}_#{route[:route_method]}_path"
+                 elsif route[:namespace].nil? && route[:route_method].present?
+                   "#{route[:route_method]}_path"
+                 end
+               else
+                 "#{name.to_s.pluralize.parameterize.underscore}_path"
+               end
+
+      Rails.application.routes.url_helpers.public_send helper
```

Comment:
```rb
let(:helper) do
  return "#{name.to_s.pluralize.parameterize.underscore}_path" if route.nil?
  
  if route[:namespace].present?
    route[:route_method].present? ? "#{route[:namespace]}_#{route[:route_method]}_path" :  "#{route[:namespace]}_#{name.to_s.pluralize.parameterize.underscore}_path"
  else
    "#{route[:route_method]}_path" if route[:route_method].present?
  end
end

let(:expected_route) { Rails.application.routes.url_helpers.public_send(helper) }
```

What do you think about splitting let?

Reply:
Thanks! I've rewritten the tests here. Now, it's more beautiful, I think :slightly_smiling_face: